src/main/java/nl/dtls/fairdatapoint/entity/settings/Settings.java

```diff
@@ -58,10 +62,12 @@ public class Settings {
 
     public static Settings getDefault() {
         return Settings.builder()
+                .appTitle(null)
+                .appSubtitle(null)
                 .metadataMetrics(DEFAULT_METRICS)
                 .ping(SettingsPing.builder()
                         .enabled(true)
-                        .endpoints(List.of("https://home.fairdatapoint.org"))
+                        .endpoints(Collections.emptyList())
```

Comment:
Default is empty in the new situation? Or is it populated from somewhere else?

Reply:
Now is the default back in config file... or should it be vice verse (empty default for config file and home.fairdatapoint.org as default in the settings)?

Reply:
That will have to depend on the default behaviour. When first booting, the FDP will ping home. When configured in the config file, this can be set before first ping. Is this correct?

Reply:
Yes, exactly... you can change only the file before the first startup.

Technically, you could somehow initiate the DB before but that is not standard use case I guess 😄